### PR TITLE
Fix setup.py for installation with zip file

### PR DESCRIPTION
**Pull request by titilambert**
Hello !
This is a small fix to be able to use ```python setup.py install``` outsite a git repository
To reproduce this problem:
```
git clone https://github.com/naparuba/shinken.git
cd shinken
rm -rf .git
python setup.py install
```

This patch fix it :)

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -1,7 +1,13 @@
 #!/usr/bin/python
+import os
+from shinken.bin import VERSION
```
Comment (naparuba) [inline setup.py]: How can it import shinken if it's not already installed or in the PYTHONPATH?

Comment (Seb-Solon) [inline setup.py]: Hum, in this case this I think python is doing a relative import . Setup.py is next to the shinken directory in the repo

**Comment by titilambert** [comment]
This works for me. Could you confirm it on your env ?
Thanks !

**Comment by gst** [comment]
> This works for me. Could you confirm it on your env ?

`python setup.py install` works for me too in this branch, modulo the pre-creation of a virtualenv.. (otherwise you missed sudo for a system install).

**Pull request merged**